src/setup_node_env/node_version_validator.test.js

```diff
@@ -68,4 +68,30 @@ describe('NodeVersionValidator', function () {
       done();
     });
   });
+
+  it('should run the script WITHOUT error when only the patch version is different', function (done) {
+    var matches = REQUIRED_NODE_JS_VERSION.match(/^v(\d+)\.(\d+)\.(\d+)/);
+    var major = matches[1];
+    var minor = matches[2];
+    var patch = parseInt(matches[3]) + 1; // change patch version to be higher than required
+
+    var processVersionOverwrite =
+      "Object.defineProperty(process, 'version', { value: '" +
+      'v' +
+      major +
+      '.' +
+      minor +
+      '.' +
+      patch +
+      "', writable: true });";
```

Comment:
nit: can we use a template literal here to make this slightly more readable?

Reply:
Fixed in 199b8c4d43